From fafd942c1376a95265123afb8d80cb9cb5f2ddda Mon Sep 17 00:00:00 2001
From: Xinyun Liu <xinyun.liu@intel.com>
Date: Wed, 27 Jun 2018 16:07:58 +0800
Subject: [PATCH 1572/2328] Revert "drm/i915/gvt: hard code Pipe B plane owner
 to UOS"

This reverts commit 47ef8f03a3d54c33bc2d8be69dee50e8dd8bb7cc.

plane restriction feature is merged and this hard code should be
removed.

Signed-off-by: Xinyun Liu <xinyun.liu@intel.com>
Reviewed-by: Min He <min.he@intel.com>
Reviewed-by: Fei Jiang <fei.jiang@intel.com>
---
 drivers/gpu/drm/i915/gvt/acrn-gvt.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/gpu/drm/i915/gvt/acrn-gvt.c b/drivers/gpu/drm/i915/gvt/acrn-gvt.c
index 1d629f8..96f5070 100644
--- a/drivers/gpu/drm/i915/gvt/acrn-gvt.c
+++ b/drivers/gpu/drm/i915/gvt/acrn-gvt.c
@@ -406,12 +406,6 @@ static int acrngt_sysfs_add_instance(struct acrngt_hvm_params *vp)
 
 	struct intel_vgpu_type type = acrngt_priv.gvt->types[0];
 
-	/* todo: wa patch due to plane restriction patches are not porting */
-	acrngt_priv.gvt->pipe_info[1].plane_owner[0] = 1;
-	acrngt_priv.gvt->pipe_info[1].plane_owner[1] = 1;
-	acrngt_priv.gvt->pipe_info[1].plane_owner[2] = 1;
-	acrngt_priv.gvt->pipe_info[1].plane_owner[3] = 1;
-
 	type.low_gm_size = vp->aperture_sz * VMEM_1MB;
 	type.high_gm_size = (vp->gm_sz - vp->aperture_sz) * VMEM_1MB;
 	type.fence = vp->fence_sz;
-- 
2.7.4

